include/nlohmann/detail/input/json_sax.hpp

```diff
@@ -548,6 +696,61 @@ class json_sax_dom_callback_parser
     }
 
   private:
+
+    HANDLE_START_END_POS_DEFINITION(json_value,
+    {
+        auto* v_base = reinterpret_cast<::nlohmann::detail::json_base_class_with_start_end_markers*>(&v);
+        v_base->end_position = m_lexer_ref->get_position();
+
+        switch (v.type())
+        {
+            case value_t::boolean:
+            {
+                v_base->start_position = v_base->end_position - (v.m_data.m_value.boolean ? 4 : 5);
```

Comment:
Again, please add a comment for the magic numbers.

Reply:
Added